src/client/search/SearchActions.js

```diff
@@ -22,3 +25,24 @@ const addQueryParamTypes = (searchQuery, query, paramTypes) => {
     paramTypes,
   };
 };
+
+export const INVALIDATE_FEED_RESULTS = 'INVALIDATE_FEED_RESULTS';
+export const invalidateFeedResults = () => {
+  return (dispatch, getState) => {
+    dispatch({ 'type': INVALIDATE_FEED_RESULTS });
+    dispatch(searchApiForFeed(getState().searchTerms));
+  };
+};
+
+export const UPDATE_FEED_RESULTS = 'UPDATE_FEED_RESULTS';
```

Comment:
This isn't a request though, this is saying "We've got the results ready, place them in the feed" so update makes sense to me?
Any other verb suggestions?